docs/porting/psa/TLS-on-PSA.md

```diff
@@ -14,9 +14,7 @@ If an application was built with Mbed TLS that was not PSA enabled, you can stil
 
 _Please note that this is likely to change at a later date though, as existing interfaces are likely to be deprecated in favor of the new, PSA-based APIs - if, when and how it happens is outside the scope of this document._
 
-If your application doesn't use any of the PSA-specific APIs described below, then you can use it with the version in Mbed OS 5.12 by just recompiling it. (Note that relinking is not enough, though.)
-
-If your application uses one of the PSA-specific APIs described below, then you need to change its source code because the APIs have changed in an incompatible way between Mbed OS 5.11 and Mbed OS 5.12 to reflect changes in PSA Crypto, whose API wasn't stable as of Mbed OS 5.11.
+If your application doesn't use any of the PSA-specific APIs described below, you can use it with the version in Mbed OS by just recompiling it. (Note that relinking is not enough, though.)
```

Comment:
why that "though" at the end?

Reply:
Well, it was always there so I didn't touch it :) It does make sense, grammatically, but maybe not technically. 

Can you explain what this is trying to tell the reader? For example, why "relinking", which isn't a word that was used anywhere else?

Reply:
I got no idea what it is trying to say. Best to ask @Patater .

Reply:
Hey @Patater can you help us out? What is "Note that relinking is not enough, though" mean? Relinking isn't used earlier in the paragraph. Did you mean "you can't just relink, you must recompile"?

Reply:
Since this isn't new, I'm going to merge this PR and look into the PSA relinking question as a new task